scripts/files/503-no-webcam.http

```diff
@@ -0,0 +1,111 @@
+                padding: 2px 4px;
+                white-space: nowrap;
+                background-color: #f7f7f7;
+                border: 1px solid #e1e1e8;
+            }
+
+            @media (max-width: 767px) {
+                .wrapper {
+                    padding: 20px;
+                }
+            }
+
+            @media (min-width: 768px) {
+                .wrapper {
+                    position: absolute;
+                    width: 750px;
+                    height: 600px;
+                    top: 50%;
+                    left: 50%;
+                    margin: -300px 0 0 -375px;
+                }
+            }
+
+            h1 {
+                line-height: 1.3;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="wrapper">
+            <h1>The webcam server is currently not running</h1>
+
+            <p><strong>
+                If you do not have a camera attached, this is normal and can be safely ignored.
+            </strong></p>
+
+            <p>
+                Otherwise, if you just started up your Raspberry Pi or just plugged in your camera,
+                please wait a couple of seconds.
+            </p>
+
+            <p>
+                If the issue persists, please check the following:
+            </p>
+
+            <ul>
+                <li>
+                    If you have a Raspberry Pi camera, verify that it is properly attached. The ribbon
+                    cable can be plugged in the wrong way. <strong>Power off your Pi first, do not attempt
+                    to attach or detach the Raspberry Pi camera while the Pi is powered!</strong>
+                </li>
+                <li>
+                    Your camera might not be configured correctly. Please check the configuration files in <code>/boot/camera-streamer/</code> and
+                    make sure they are correct for your camera. See <a href="https://faq.octoprint.org/camera-streamer-config">the configuration documentation</a>
+                    for more info.
+                </li>
+                <li>
+                    Log into your Raspberry Pi via SSH. Check if your <strong>USB</strong> camera is detected by the system via <code>lsusb</code>.
+                </li>
+                <li>
+                    Log into your Raspberry Pi via SSH. Check if your <strong>Raspberry Pi camera</strong> is detected by the system via
+                    <code>libcamera-hello --list-cameras</code>.
+                </li>
+                <li>
+                    If your camera is detected, check what the camera stack is reporting by running <code>journalctl -u camera-streamer\* | pb</code>. You can use the resulting
+                    <a href="https://paste.octoprint.org">paste.octoprint.org</a> link to view the logs or share them with the community easily.
+                    You can also get these logs by downloading a systeminfo bundle from your OctoPrint server.
```

Comment:
This is an assumption that this will be implemented in the Pi Support plugin at some point.

Reply:
Yeah, that was the goal, just haven't gotten around to it yet.